drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableViewImpl.java

```diff
@@ -110,9 +110,27 @@ public GuidedDecisionTableViewImpl(final GridData uiModel,
         this.access = access;
         this.headerCaption = makeHeaderCaption();
 
+        registerNodeDragMoveHandler();
+        registerNodeMouseDoubleClickHandler();
+    }
+
+    void registerNodeDragMoveHandler() {
         addNodeDragMoveHandler((event) -> presenter.getModellerPresenter().updateRadar());
     }
 
+    void registerNodeMouseDoubleClickHandler() {
```

Comment:
I miss some test for the logic of the double click handler. Ideally I would move the handler logic into the presenter class. It will probably require to make the `isNodeMouseEventOverCaption` as public method of the view.

I admit that the logic is not complex but I think worth to test. If you are not going to add the test please file a jira for me @manstis